packages/graphile-build-pg/src/QueryBuilder.d.ts

```diff
@@ -93,6 +93,14 @@ export default class QueryBuilder {
     addNotDistinctFromNullCase?: boolean;
     useAsterisk?: boolean;
   }): SQL;
+  public buildChild(): QueryBuilder;
+  public buildNamedChildFrom(
+    name: string,
+    from: SQLGen,
+    field: SQLGen,
+    alias: SQLAlias
```

Comment:
I think this can be optional:

```suggestion
    alias?: SQLAlias
```